Realm/RLMAccessor.mm

```diff
@@ -597,6 +597,21 @@ void RLMReplaceSharedSchemaMethod(Class accessorClass, RLMObjectSchema *schema)
     class_replaceMethod(metaClass, @selector(sharedSchema), imp, "@@:");
 }
 
+// Declare a selector that is only implemented by generated classes
+@interface RLMDummyDeclaration : NSObject
++ (void)RLMIsGeneratedClass;
+@end
+
+static void RLMMarkClassAsGenerated(Class cls) {
```

Comment:
Why not store a vector of generated classes instead? Then you could check for its presence in the vector in `RLMIsGeneratedClass` without the need to use the Objective-C runtime or the dummy class & selector.